frontend/src/pages/modelServing/screens/projects/kServeModal/ManageKServeModal.tsx

```diff
@@ -145,28 +146,27 @@ const ManageKServeModal: React.FC<ManageKServeModalProps> = ({
       editInfo?.inferenceServiceEditInfo?.spec.predictor.model.runtime ||
       translateDisplayNameForK8s(createDataInferenceService.name);
 
-    Promise.all([
-      submitServingRuntimeResources(
-        servingRuntimeSelected,
-        createDataServingRuntime,
-        customServingRuntimesEnabled,
-        namespace,
-        editInfo?.servingRuntimeEditInfo,
-        true,
-        acceleratorProfileState,
-        NamespaceApplicationCase.KSERVE_PROMOTION,
-        projectContext?.currentProject,
-        servingRuntimeName,
-        false,
-      ),
-      submitInferenceServiceResource(
-        createDataInferenceService,
-        editInfo?.inferenceServiceEditInfo,
-        servingRuntimeName,
-        false,
-        acceleratorProfileState,
-      ),
-    ])
+    submitServingRuntimeResources(
```

Comment:
Yes, that's the solution I was thinking about for https://issues.redhat.com/browse/RHOAIENG-556, you've solved it in this PR, so attatch the link in the pr description and update the jira issue accordingly, great work here!

Reply:
Done!